source/core/NumericString.cpp

```diff
@@ -2715,6 +2800,8 @@ DEFINE_VIREO_BEGIN(NumericString)
     DEFINE_VIREO_FUNCTION(StringFormat, "p(i(VarArgCount) o(String)   i(String) io(ErrorCluster err) i(StaticTypeAndData))")
     DEFINE_VIREO_FUNCTION(StringScanValue, "p(i(String) o(String) i(String) o(StaticTypeAndData))")
     DEFINE_VIREO_FUNCTION(StringScan, "p(i(VarArgCount) i(String) o(String) i(String) i(UInt32) o(UInt32) io(ErrorCluster err) o(StaticTypeAndData))")
+    DEFINE_VIREO_FUNCTION(ByteArrayToString, "p(o(String) i(Array) i(Int32) io(ErrorCluster err))")
```

Comment:
Please update the order of parameters, inputs first, outputs last, and also update the param block declaration

DEFINE_VIREO_FUNCTION(ByteArrayToString, "p(i(Array) i(Int32) o(String) io(ErrorCluster err))")
DEFINE_VIREO_FUNCTION(StringToByteArray, "p(i(String) i(Int32) o(Array) io(ErrorCluster err))")